packages/client/src/components/app/forms/S3Upload.svelte

```diff
@@ -2,7 +2,7 @@
   import Field from "./Field.svelte"
   import { CoreDropzone, ProgressCircle } from "@budibase/bbui"
   import { getContext, onMount, onDestroy } from "svelte"
-  import { cloneDeep } from "../../../../../bbui/src/helpers"
+  import { cloneDeep } from "@budibase/bbui/src/helpers"
```

Comment:
is this one ok?

Reply:
Yup, as it is not build via esbuild. But I just cleaned it :)